src/plugins/data/public/ui/filter_bar/filter_item.tsx

```diff
@@ -286,6 +286,11 @@ export function FilterItem(props: FilterItemProps) {
       message: '',
       status: FILTER_ITEM_OK,
     };
+
+    if (filter.meta?.isMultiIndex) {
```

Comment:
@Dosant @lizozom isMultiIndex flag was added to remove "Warning" label from the filter pill

The screen shot below was taken before these changes. Notice the "Warning" label. In this case, its not warranted since the filter is written to target any index pattern in the dashboard
<img width="200" alt="Screen Shot 2021-05-12 at 8 04 38 AM" src="https://user-images.githubusercontent.com/373691/117988989-2c9cb380-b2f9-11eb-8d1e-9dced2ecdea0.png">

This is a screen shot after the changes. Notice that the warning label is gone
<img width="200" alt="Screen Shot 2021-05-12 at 8 05 00 AM" src="https://user-images.githubusercontent.com/373691/117989269-68d01400-b2f9-11eb-81c9-045f26b7c194.png">